pir/common/shard.go

```diff
@@ -1,4 +1,4 @@
-package pir
+package common
```

Comment:
Can we rename this package something else? Like pircommon, pirinterface, pirgeneric, etc?